storage/tests/system.py

```diff
@@ -952,3 +952,17 @@ def test_notification_w_user_project(self):
             self.assertEqual(notifications[0].topic_name, self.TOPIC_NAME)
         finally:
             notification.delete()
+
+
+class TestAnonymousClient(unittest.TestCase):
+
+    PUBLIC_BUCKET = 'gcp-public-data-landsat'
+
+    def test_access_to_public_bucket(self):
+        anonymous = storage.Client.create_anonymous_client()
+        bucket = anonymous.bucket(self.PUBLIC_BUCKET)
+        for blob in bucket.list_blobs():
+            print("Downloading blob: {}".format(blob.name))
+            with tempfile.TemporaryFile() as stream:
+                blob.download_to_file(stream)
```

Comment:
@tseaver The test should verify **something** other than "we didn't fail"?

Reply:
The only interesting thing is that we can list objects and download them: there is nothing special about the data to be asserting.

Reply:
Well if the data is deterministic, we can just assert the content is what we expect.